### ISSUE TITLE
Feature Suggestion: Accessible Accordion

### ISSUE THREAD

**Issue by tbusillo** [WPDevelopers/essential-addons-for-elementor-lite#513]
This is a feature suggestion and proposed solution to make the Advanced Accordion WCAG 2.1.1 (Keyboard) compliant. There are currently two issues with the existing Accordion plugin preventing compliance: 1) The accordion elements do not receive focus, and 2) even if it did receive focus, the JQuery click event handler for showing/collapsing the accordion contents cannot be triggered via keypress (e.g., enter key or space bar). 

Here is the proposed solution: 

1) Replace Adv_Accordion.php, line 1024 with: 

`echo '<div class="eael-accordion-list" tabindex="0">`

2) Trigger the existing click handler in advanced-accordion.js using something like: 

```
jQuery(document).ready(function($){
	$(".eael-accordion-header").on('keypress', function(e) {
		var id = $(this).attr('id');
		e.preventDefault();
		if(e.which == 13 || e.which == 32) {
			$('#'+id).trigger("click");
		}
	})
});
```
If you were to do it within advanced-accordion.js, it would probably look something like: 

```
    $accordionHeader.unbind("keypress");

    $accordionHeader.on("keypress", function(e) {
        var id = $(this).attr('id');
        e.preventDefault();
        if(e.which == 13 || e.which == 32) {
            $('#'+id).trigger("click");
        }
    });
```

This solution isn't the most elegant and there are ways it could be better implemented within the existing EA code. I'm not familiar enough with the codebase to submit a pull request. However, the solution above has been tested and it successfully enables the ability to use the "enter" (keycode 13) or "space" (keycode 32) to trigger the click handler in Adv_Accordion.php. 

I hope this helps!

**Comment by IftekharPial** [comment]
Hello there @tbusillo 

Hope you are doing well. Sorry for the delayed response.

You will be glad to know that we have introduced Accessibility support to the EA Advanced Accordion.

Please update to the latest version of Essential Addons **v5.8.6** to get access to the feature.

Note: Please Regenerate the asset after updating to the latest version
To regenerate assets, you can go to wp-admin -> Essential Addons then, switch to the ‘Tools’ tab and hit the ‘Regenerate Assets’ button. Check out this screenshot: https://d.pr/i/FuLkYy .If you are using a caching plugin, you need to purge the cache from your plugin option and then deactivate the caching plugin temporally so that the new update could take effect.

Hope it helps
Let me know if you face any issues
Thanks
